src/modals/LedgerImport/Addresses/Wrappers.tsx

```diff
@@ -97,6 +97,12 @@ export const AddressWrapper = styled.div`
             h5,
             button {
               font-size: 0.9rem;
+              &.label {
+                font-size: 0.85rem;
+                display: flex;
+                flex-flow: row wrap;
```

Comment:
What happens if you remove `flex-flow: row wrap` from each of these classes? The default value is `flex-flow: row nowrap`, I am guessing that we probs do not need wrapping on these labels (we don't want them to go multi line?).

Reply:
You are right, I will remove the line